header.js

```diff
@@ -0,0 +1,11 @@
+module.exports = `!
+ * Copyright (c) 2015-present, Okta, Inc. and/or its affiliates. All rights reserved.
```

Comment:
Can we make this dynamic? Usually this adheres to the format `${yearFileCreated}-Present`.

Reply:
yea I can - but it will limit the ability to use the `--fix` eslint flag I imagine. Will have to do some more investigation

Reply:
was able to get this working in latest patch here